alerting/src/main/java/org/hawkular/metrics/alerting/InsertedDataSubscriber.java

```diff
@@ -129,10 +132,33 @@ private void publishNumeric(Metric<? extends Number> numeric) {
         }
     }
 
+    private String getDataId(MetricId<? extends Number> numericId) {
```

Comment:
could be done, if I need to change anything else in there I can do it to make you happy :)